numba/tests/npyufunc/test_cuda_gufunc.py

```diff
@@ -0,0 +1,190 @@
+from __future__ import print_function, absolute_import
```

Comment:
You could still group them as ufunc tests inside the CUDA tests directory ;) The separation between CPU and GPU tests right now is very readable, it becomes less so if we start putting CUDA tests in the main tests directory.